lib/shopify_app/carrier_services_manager_job.rb

```diff
@@ -0,0 +1,10 @@
+module ShopifyApp
+  class CarrierServicesManagerJob < ActiveJob::Base
+    def perform(shop_domain:, shop_token:)
+      ShopifyAPI::Session.temp(shop_domain, shop_token) do
+        manager = CarrierServicesManager.new
+        manager.create_carrier_services
```

Comment:
I was just working on the webhook manager and I noticed from reading the code that I think `create_*` already does what I describe (at least for webhooks). It tries to create all the required hooks - if we get an error we check if the webhook is persisted already and if it is we continue without raising and error. This might be unique to how webhooks because of topic validations so we should confirm still and this probably doesn't apply directly to carrier services